### ISSUE TITLE
Add rule to flag multiple inputs in a single label

### ISSUE THREAD

**Issue by dylanb** [dequelabs/axe-core#1053]
According to the HTML spec: a `<label>` element may contain at most one`<button>`, `<input>`, `<meter>`, `<output>`, `<progress>`, `<select>`, or `<textarea>` descendant.

Having multiple causes accessibility issues so we should add a rule to flag this under 4.1.1

**Comment by WilcoFiers** [comment]
@dylanb I don't think this is a violation of 4.1.1. WCAG doesn't require content to be compliant to the HTML spec. Beyond just checking that form fields have an accessible name, which axe-core already does, I don't see how this is a different issue. Is there any case where both element have an accessible name that would be a violation of WCAG 2 SC 4.1.1.?

I would say the following is fine for 4.1.1 and 4.1.2. But it doesn't pass the [3.2.2 labels or instructions](https://www.w3.org/TR/WCAG21/#labels-or-instructions).

```html
<label>Phone number <input type="text" size="3" /> - <input type="text"></label>
```

The way I think we should handle this is to create a rule that checks that within a form / fieldset forms have a unique name, similar to how we check unique titles for iframes in pages.

**Comment by dylanb** [comment]
ok, I am ok with adding the rule under 3.2.2

**Comment by WilcoFiers** [comment]
This was done a while ago in the `form-field-multiple-labels` rule.
